mol_dqn/chemgraph/configs/bootstrap_dqn.json

```diff
@@ -0,0 +1,33 @@
+{
+  "atom_types": ["C", "O", "N"],
+  "max_steps_per_episode": 40,
+  "allow_removal": false,
+  "allow_no_modification": true,
+  "allow_bonds_between_rings": false,
+  "allowed_ring_sizes": [5, 6],
+  "replay_buffer_size": 5000,
+  "learning_rate": 0.0001,
+  "learning_rate_decay_steps": 10000,
+  "learning_rate_decay_rate": 0.9,
+  "num_episodes": 5000,
+  "batch_size": 128,
+  "learning_frequency": 4,
+  "update_frequency": 20,
+  "grad_clipping": 10,
+  "gamma": 0.8,
+  "discount_factor": 1,
```

Comment:
This should be a float, not an int (same for all configs).

Reply:
Done